.github/workflows/build_test.yml

```diff
@@ -202,18 +202,18 @@ jobs:
         python-version: ${{ env.PYTHON_VERSION }}
     - uses: ./.github/actions
 
-  pypy37:
-    needs: [smoke_test]
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: recursive
-        fetch-depth: 0
-    - uses: actions/setup-python@v2
-      with:
-        python-version: pypy-3.7-v7.3.4
-    - uses: ./.github/actions
+  #pypy37:
+    #needs: [smoke_test]
+    #runs-on: ubuntu-latest
+    #steps:
+    #- uses: actions/checkout@v2
+      #with:
+        #submodules: recursive
+        #fetch-depth: 0
+    #- uses: actions/setup-python@v2
+      #with:
+        #python-version: pypy-3.7-v7.3.4
+    #- uses: ./.github/actions
```

Comment:
Hopefully we can turn this back on with pypy-3.8 soonish

Reply:
Yes, that's why I commented it out instead of deleting.